src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Properties/AssemblyInfo.cs

```diff
@@ -8,12 +8,12 @@
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services SpellCheck API.")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
In case of major version update, AssemblyVersion must also be updated. Apologize for the back and forth